integration/auth/auth_test.go

```diff
@@ -260,6 +267,198 @@ func signInWithPassword(email, password string) (string, error) {
 	return respBody.IDToken, err
 }
 
+func TestSaltPasswordOrder(t *testing.T) {
```

Comment:
Let's verify this behavior in the unit tests. That's faster and also gets checked more often in our CI. In the integration tests we can check one hash configuration with input order for completeness. Also add it in the `uset_mgt_test.go` file closer to the other `ImportUsers()` tests.

Reply:
The purpose of all of this is to make sure that the remote server is properly processing the options I gave it, being responsive to whether the password or hash should come first.  Isn't that the definition of an integration test?

Reply:
Testing if the remote server works correctly is out of scope for us. Our testing should cover the correct serialization of the arguments and the HTTP request/response handling, which can be done easily in the unit tests.